src/util/mod.rs

```diff
@@ -125,7 +125,7 @@ pub(crate) async fn local_interfaces(
     ips
 }
 
-pub(crate) async fn listen_udp_in_port_range(
+pub async fn listen_udp_in_port_range(
```

Comment:
Why there are several changes like this?

Reply:
pub(crate)=>pub?

Reply:
It was reported as [redundant pub crate](https://rust-lang.github.io/rust-clippy/master/index.html#redundant_pub_crate).

If the `mod` is private, then the `fn` wouldn't be visible outside the crate. Now that I think about it, If we changed the visibility of the mod, it would reflect on it. I can revert it back if that's the plan.

Reply:
got it. In this case, we don't need to revert it back. just follow clippy suggestion.